frontend/services/entities/software.ts

```diff
@@ -234,7 +234,37 @@ export default {
       true
     );
   },
+  editSoftwarePackage: (
+    softwareId: number,
+    data: IAddPackageFormData, // TODO: Update name in renaming
+    teamId?: number,
+    timeout?: number
+  ) => {
+    const { SOFTWARE_TITLE } = endpoints;
+
+    if (!data.software) {
+      throw new Error("Software package is required");
+    }
 
+    const formData = new FormData();
+    formData.append("software", data.software);
+    formData.append("self_service", data.selfService.toString());
+    data.installScript && formData.append("install_script", data.installScript);
+    data.preInstallQuery &&
+      formData.append("pre_install_query", data.preInstallQuery);
+    data.postInstallScript &&
+      formData.append("post_install_script", data.postInstallScript);
+    teamId && formData.append("team_id", teamId.toString());
+
+    return sendRequest(
+      "POST",
+      SOFTWARE_TITLE(softwareId),
+      formData,
+      undefined,
```

Comment:
Per the API, we're returning the updated software title here, and I'm re-pulling the title using the same logic I'd use for a GET. Would it make sense to use it on the frontend from here, or am I misunderstanding what `undefined` is doing here?

Reply:
Hey @iansltx - Current draft is reflecting that I haven't put much thought into how the API call works, adding it as a todo for later today. Will respond then. <3

TODO: @RachelElysia

Reply:
TODO: @RachelElysia only send changing fields.